opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.hpp

```diff
@@ -122,7 +122,7 @@ class EndpointInitializer
             for (int rowIdx = 0; rowIdx < numRows; ++rowIdx) {
                 if (krwCol[rowIdx] > 0.0) {
                     double Sw = 0.0;
-                    if (rowIdx > 0)
+                    if (rowIdx > 0.0)
```

Comment:
ehem, yes it is. fixed & thanks. (for some reason I didn't see a compiler warning, though.)